src/main.cpp

```diff
@@ -981,6 +981,36 @@ bool ContextualCheckTransaction(
             // https://zips.z.cash/zip-0213#specification
             uint256 ovk;
             for (const OutputDescription &output : tx.vShieldedOutput) {
+                bool zip_212_enabled;
+                libzcash::SaplingPaymentAddress zaddr;
+                CAmount value;
+
+            // 5.4.2 EoS height is 2121024.
```

Comment:
```suggestion
            // 5.3.3 and 5.4.2 EoS height is 2121024.
```
Also, I checked this height.